app/viewmodels/checkAnswers/ReturnPeriodSummary.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers
+
+import controllers.routes
+import models.{CheckMode, UserAnswers}
+import play.api.i18n.Messages
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object ReturnPeriodSummary {
+
+  def row(userAnswers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] = {
+    Some(SummaryListRowViewModel(
+      key = "checkYourAnswers.returnPeriod.checkYourAnswersLabel",
+      value = ValueViewModel(HtmlFormat.escape(userAnswers.period.displayText).toString),
+      actions = Seq(
+        ActionItemViewModel("site.change", routes.StartReturnController.onPageLoad(userAnswers.period).url)
+          .withVisuallyHiddenText(messages("countryOfConsumptionFromNi.change.hidden"))
```

Comment:
We need to go back over and update message keys for visiually hidden text on change links